metadata/manager.go

```diff
@@ -453,3 +455,7 @@ components + {
 }
 `)
 }
+
+func errorOnCreateFailure(appName string, err error) error {
+	return fmt.Errorf("%s\nTo undo this simply delete directory '%s' and re-run `ks init`. If error persists, try using flag '--context' to set a different context or run `ks init --help` for more options", err, appName)
```

Comment:
suggestion: put a newline after "re-run `ks init`".
grammar nit: s/If error/If the error